tutor/src/components/payments/manage.jsx

```diff
@@ -53,6 +123,12 @@ export default class ManagePayments extends React.PureComponent {
           <BackButton fallbackLink={this.backLink} />
         </header>
         {Purchases.hasApiRequestPending ? <OXFancyLoader isLoading /> : this.renderTable()}
+        <div className="refund-policy">
+          <NewTabLink to="http://openstax.force.com/support/articles/FAQ/OpenStax-Tutor-Student-Refund-Policy/?q=refund&l=en_US&c=Products%3ATutor&fs=Search&pn=1">
```

Comment:
i kinda think this, along with some other rando links that we have like this, would be nice to pull out into a constant for easy updates later on, but not a big deal for now